raxx_elli/lib/raxx/elli/handler.ex

```diff
@@ -30,10 +30,19 @@ defmodule Raxx.Elli.Handler do
     headers = :elli_request.headers(elli_request) |> Enum.map(fn ({k, v}) ->
       {String.downcase(k), String.downcase(v)}
     end)
-    {"Host", authority} = :elli_request.headers(elli_request) |> List.keyfind("Host", 0)
-    [host, port] = String.split(authority, ":")
+
+	[host, port] = 
```

Comment:
I hope I'm not starting a tabs v spaces argument but this should have the same indentation as the lines deleted